packages/api-key-stamper/src/utils.ts

```diff
@@ -11,7 +11,14 @@ export function convertTurnkeyApiKeyToJwk(input: {
 }): JsonWebKey {
   const { uncompressedPrivateKeyHex, compressedPublicKeyHex } = input;
 
-  const jwk = pointDecode(uint8ArrayFromHexString(compressedPublicKeyHex));
+  let jwk;
+  try {
+    jwk = pointDecode(uint8ArrayFromHexString(compressedPublicKeyHex));
+  } catch (e) {
+    throw new Error(
+      `invalid API key: Ensure that you are using a valid public and private key in compressed or uncompressed format and they are not switched`
```

Comment:
Do you want to template the value of `compressedPublicKeyHex` in this case?

I think this message should be purely about API public key rather than both public and private keys. Something like `unable to load API key: invalid public key ("${compressedPublicKeyHex}")`?